src/test/java/com/wikia/webdriver/testcases/mobile/HTMLTitleTests.java

```diff
@@ -32,22 +32,22 @@
    * [0] wikiName [1] path [2] expected title
    */
   private String[][] testCases = {
-      {"sktest123", "Sktest123_Wiki", "Sktest123 Wiki | FANDOM powered by Wikia"},
-      {"sktest123", "Style-5H2", "Style-5H2 | Sktest123 Wiki | FANDOM powered by Wikia"},
-      {"sktest123", "TestDisplayTitle", "testing abc | Sktest123 Wiki | FANDOM powered by Wikia"},
-      {"sktest123", "Category:Premium_Videos",
+      {"sktest123", "en", "Sktest123_Wiki", "Sktest123 Wiki | FANDOM powered by Wikia"},
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "sktest123" 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)